fuel/app/classes/materia/widget/instance.php

```diff
@@ -14,6 +14,7 @@ class Widget_Instance
 	public $embed_url       = '';
 	public $is_student_made = false;
 	public $is_embedded     = false;
+	public $is_deleted			= false;
```

Comment:
spaces between stuff, tabs to indent
```suggestion
	public $is_deleted      = false;
```